org.KangLin.Rabbit.RemoteControl.json

```diff
@@ -0,0 +1,262 @@
+/* See: https://docs.flatpak.org/en/latest/manifests.html */
+/* See: https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html#flatpak-manifest */
+{
+    "id": "org.KangLin.Rabbit.RemoteControl",
+    /*See: https://invent.kde.org/packaging/flatpak-kde-runtime*/
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.8",
+    "sdk": "org.kde.Sdk",
+    "command": "RabbitRemoteControlApp",
+    "add-build-extensions": {
+        "org.freedesktop.Platform.ffmpeg-full": {
+            "directory": "lib/ffmpeg",
+            "version": "23.08",
+            "add-ld-path": "."
+        },
+        "org.freedesktop.Platform.openh264": {
+            "directory": "lib/openh264",
+            "version": "2.4.1",
+            "add-ld-path": "."
+        }
+    },
+    "build-options": {
+        "env": {
+            /*"V": "1"*/
+        }
+    },
+    "cleanup": [
+        "*.la",
+        "*.a",
+        "/include",
+        "/lib/cmake",
+        "/lib/pkgconfig"
+    ],
+    "finish-args": [
+        /* Needs to talk to the network */
+        "--share=network",
+        /* Play sounds redirected from guests */
+        "--socket=pulseaudio",
+        /* X11 + XShm access */
+        "--socket=x11",
+        /* force use X11 */
+        /*"--env=QT_QPA_PLATFORM=xcb",*/
+        "--socket=wayland",
+        "--device=all",
```

Comment:
why device `all` ?

Reply:
need camera, fuse, audio,video, usb. so set all.

Reply:
this is where comments are useful.